libsignal/src/main/java/org/session/libsignal/utilities/HTTP.kt

```diff
@@ -15,7 +19,13 @@ object HTTP {
     var isConnectedToNetwork: (() -> Boolean) = { false }
 
     private val seedNodeConnection by lazy {
-        OkHttpClient().newBuilder()
+        val bootstrapClient = OkHttpClient.Builder().build()
+        val dns =DnsOverHttps.Builder().client(bootstrapClient)
+            .url("https://dns.quad9.net/dns-query".toHttpUrl())
+            .bootstrapDnsHosts(InetAddress.getByName("8.8.4.4"), InetAddress.getByName("8.8.8.8"))
```

Comment:
do these need to be updated to `9.9.9.9` ?

Reply:
and maybe `9.9.9.11` as the second one

Reply:
If the user's network cannot use dns.quad9.net,  fall back to 8.8.8.8
Because dns.quad9.net cannot be used, nor can 9.9.9.9

Reply:
Maybe I'm not too sure how the DnsBuilder will use bootStrapDnsHosts, you mean it will query the 8.8.4.4 and 8.8.8.8 to get the quad9 dns resolver or it will use those IPs if the url supplied is unreachable?

Reply:
first use quad9, if not available,use 8.8.8.8 8.8.4.4